sensing/imu_corrector/test/test_gyro_bias_estimation_module.cpp

```diff
@@ -63,7 +63,9 @@ TEST_F(GyroBiasEstimationModuleTest, GetBiasEstimationWhenVehicleStopped)
 
 TEST_F(GyroBiasEstimationModuleTest, GetInsufficientDataException)
 {
-  ASSERT_THROW(module.get_bias_base_link(), std::runtime_error);
+  // for the case of method with [[nodiscard]] attribute
+  geometry_msgs::msg::Vector3 bias_base_link;
+  ASSERT_THROW(bias_base_link = module.get_bias_base_link(), std::runtime_error);
```

Comment:
This is for https://github.com/google/googletest/issues/2416 .
Not sure with the solution.